src/shared/lib/StoreUtils.ts

```diff
@@ -97,12 +100,29 @@ export async function fetchUniprotId(swissProtAccession: string)
     return uniprotData.text.split("\n")[1];
 }
 
-export async function fetchPfamGeneData(swissProtAccession: string)
+export async function fetchPfamGraphicsData(swissProtAccession: string)
```

Comment:
Actually, we don't need pfam graphics data anymore...

Reply:
if we can delete it lets delete it